flight/waypoint/goto.py

```diff
@@ -53,16 +49,13 @@ async def move_to(
             drone_long: float = position.longitude_deg
             drone_alt: float = position.relative_altitude_m
 
+            total_distance: float = calculate_distance(
+                drone_lat, drone_long, drone_alt, latitude, longitude, altitude
+            )
             #  accurately checks if location is reached and stops for 15 secs and then moves on.
```

Comment:
Not sure where the 15 second wait is, but please reduce that as much as possible.

Reply:
It was never there, no idea why this comment says this, deleting it